recipes/cpp-sort/all/conanfile.py

```diff
@@ -25,21 +25,24 @@ def _minimum_cpp_standard(self):
         return 14
 
     @property
-    def _minimum_compilers_version(self):
+    def _compilers_minimum_version(self):
         return {
             "apple-clang": "9.4",
             "clang": "3.8",
             "gcc": "5.5",
             "Visual Studio": "16"
```

Comment:
There's a help for this one that also does the new conpiler https://github.com/conan-io/conan-center-index/blob/2fc42d1488d5bf29af9aad1c16a0f2f112db1110/docs/package_templates/cmake_package/all/conanfile.py#L90

Reply:
Oooh, nifty :o